internal/configuration/validator/notifier.go

```diff
@@ -1,11 +1,13 @@
 package validator
 
 import (
+	"errors"
 	"fmt"
 
 	"github.com/authelia/authelia/internal/configuration/schema"
 )
 
+//nolint:gocyclo // TODO: Remove in 4.28. Should be able to remove this during the removal of deprecated config.
```

Comment:
Doesn't matter since this is going to be removed in the deprecation, but try to ensure nolint directives should be specified after the GoDoc.